Library/Formula/libqxt.rb

```diff
@@ -0,0 +1,51 @@
+class Libqxt < Formula
+  desc "Extension library for Qt providing a suite of utility classes"
+  homepage "https://bitbucket.org/libqxt/libqxt/wiki/Home"
+
+  stable do
+    url "https://bitbucket.org/libqxt/libqxt/get/v0.6.2.tar.gz"
+    sha256 "582426d6c81958dd1ac5ca4a9d49807e0a2204b7343e7d49c6613bcb148b5cb8"
+    patch do
+      url "https://raw.githubusercontent.com/Homebrew/patches/master/libqxt/qxtglobalshortcut_mac.cpp.diff"
```

Comment:
Please use a short commit reference rather than `master` here; if the patch ever gets accidentally prematurely removed or changed the existing commit reference will keep working whereas `master` will break.